scripts/prepare

```diff
@@ -4,6 +4,83 @@ set -e
 # rebase the patch with upsteam charts defined in package.yaml. Make sure patches still apply
 # for the latest upsteam charts. If there is conflits commiter should solve it.
 
+function init_charts_crd_dir() {
```

Comment:
You think it would be cleaner to split this out into separate file ? ./scripts/prepare-crds.sh ...

Reply:
Yes, I think a brand new script that gets called from prepare would be easier to use. @StrongMonkey what are your thoughts on introducing this as a new script?

Reply:
Adding this as a separate file in the next commit under `scripts/prepare-crds`